.github/workflows/release-drafter-reusable.yml

```diff
@@ -0,0 +1,12 @@
+name: Re-usable Release Drafter
+
+on:
+  workflow_call:
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: release-drafter/release-drafter@v5
```

Comment:
can the reusable workflow moved to radixdlt-iac repo. So we know where to find them